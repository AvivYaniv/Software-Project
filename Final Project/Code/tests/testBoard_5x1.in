solve tests/b5_1.txt
print_board
hint 1 3
hint 2 4
undo
redo
autofill
num_solutions
reset
autofill
autofill
autofill
autofill
save tests/observed/b5x1-saveTestSolve.txt
edit tests/observed/b5x1-saveTestSolve.txt
print_board
save tests/observed/b5x1-saveTestEdit.txt
solve tests/observed/b5x1-saveTestSolve.txt
print_board