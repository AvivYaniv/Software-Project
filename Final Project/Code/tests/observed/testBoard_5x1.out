Sudoku
------
Enter your command:
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|    |    |    |  1.|  2.|
|  3.|    |    |  4 |  1 |
|  4.|    |    |    |    |
--------------------------
Enter your command:
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|    |    |    |  1.|  2.|
|  3.|    |    |  4 |  1 |
|  4.|    |    |    |    |
--------------------------
Enter your command:
Hint: set cell to 5
Enter your command:
Hint: set cell to 5
Enter your command:
Error: no moves to undo
Enter your command:
Error: no moves to redo
Enter your command:
Cell <1,3> set to 5
Cell <2,4> set to 5
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|  5 |    |    |  1.|  2.|
|  3.|  5 |    |  4 |  1 |
|  4.|    |    |    |    |
--------------------------
Enter your command:
Number of solutions: 1
This is a good board!
Enter your command:
Board reset
Enter your command:
Cell <1,3> set to 5
Cell <2,4> set to 5
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|  5 |    |    |  1.|  2.|
|  3.|  5 |    |  4 |  1 |
|  4.|    |    |    |    |
--------------------------
Enter your command:
Cell <3,3> set to 3
Cell <3,4> set to 2
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|  5 |    |  3 |  1.|  2.|
|  3.|  5 |  2 |  4 |  1 |
|  4.|    |    |    |    |
--------------------------
Enter your command:
Cell <2,3> set to 4
Cell <3,5> set to 5
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|    |  1.|    |    |
|  5 |  4 |  3 |  1.|  2.|
|  3.|  5 |  2 |  4 |  1 |
|  4.|    |  5 |    |    |
--------------------------
Enter your command:
Cell <2,2> set to 3
Cell <5,5> set to 3
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|  3 |  1.|    |    |
|  5 |  4 |  3 |  1.|  2.|
|  3.|  5 |  2 |  4 |  1 |
|  4.|    |  5 |    |  3 |
--------------------------
Enter your command:
Saved to: tests/observed/b5x1-saveTestSolve.txt
Enter your command:
--------------------------
|  1 |  2 |  4 |    |    |
|  2 |  3 |  1 |    |    |
|  5 |  4 |  3 |  1 |  2 |
|  3 |  5 |  2 |  4 |  1 |
|  4 |    |  5 |    |  3 |
--------------------------
Enter your command:
--------------------------
|  1 |  2 |  4 |    |    |
|  2 |  3 |  1 |    |    |
|  5 |  4 |  3 |  1 |  2 |
|  3 |  5 |  2 |  4 |  1 |
|  4 |    |  5 |    |  3 |
--------------------------
Enter your command:
Saved to: tests/observed/b5x1-saveTestEdit.txt
Enter your command:
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|  3 |  1.|    |    |
|  5 |  4 |  3 |  1.|  2.|
|  3.|  5 |  2 |  4 |  1 |
|  4.|    |  5 |    |  3 |
--------------------------
Enter your command:
--------------------------
|  1 |  2 |  4.|    |    |
|  2.|  3 |  1.|    |    |
|  5 |  4 |  3 |  1.|  2.|
|  3.|  5 |  2 |  4 |  1 |
|  4.|    |  5 |    |  3 |
--------------------------
Enter your command:
Exiting...
